### PR TITLE
add answersyaml field

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

**Comment by StrongMonkey** [comment]
https://github.com/rancher/rancher/issues/13158

File: apis/project.cattle.io/v3/app_types.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ type AppSpec struct {
 	Files           map[string]string `json:"files,omitempty"`
 	Answers         map[string]string `json:"answers,omitempty"`
 	AppRevisionName string            `json:"appRevisionName,omitempty" norman:"type=reference[/v3/project/schemas/apprevision]"`
-	Prune           bool              `json:"prune,omitempty"`
+	AnswersYaml     string            `json:"answersYaml,omitempty"`
```
Comment (dramich) [inline apis/project.cattle.io/v3/app_types.go]: change to ValuesYaml

**Comment by dramich** [comment]
LGTM

**Comment by cjellick** [comment]
LGTM

**Pull request merged**